foam_controlwrapper/examples/poiseuille_parallel.py

```diff
@@ -0,0 +1,62 @@
+"""Example to solve 2D poiseuille flow
+
+"""
+
+from simphony.core.cuba import CUBA
+from simphony.engine import openfoamII
```

Comment:
openfoamII is the name of the simphony.engine entry point generated by setup.py in order to use the openfoam internal interface wrapper. Currently, you can choose between "openfoam" (FILE-IO wrapper) or "openfoamII".